### ISSUE TITLE
Ski resort slope names only appear when very zoomed

### ISSUE THREAD

**Issue by yanickv** [tangrams/tangram-es#754]
Hi,

I'm trying the Android SDK to build a ski app. The issue I am having is that the ski slope names dont appear until the map is very zoomed, but then you lose the context of where the ski slope is. Is there a way to make the slope names appear sooner? (for example by tuning the yaml)

I know that the slope names are in the data received, they are just not showing (ex: https://vector.mapzen.com/osm/all/14/4798/5814.topojson?api_key=vector-tiles-xxxxxxx)

Here is an examples of Google Maps vs Tangram:

![screenshot_2016-05-24-18-23-12](https://cloud.githubusercontent.com/assets/3969320/15522977/851fd880-21e4-11e6-8324-4fbd5a78cf7f.png)

Same area with Tangram:

![screenshot_2016-05-24-18-39-07](https://cloud.githubusercontent.com/assets/3969320/15523157/63d0e794-21e6-11e6-83aa-7bf84a1c2ab8.png)

**Comment by blair1618** [comment]
Hello!

If I understand your question correctly, I think the answer is: yes, definitely! As you know, tangram uses a YAML scene file to determine which data features are drawn on the map and how they're styled. Part of this is filtering features by zoom level, like with ski slopes here. The scene files provided with the Mapzen Android SDK are large (we're trying to cover the whole planet, after all) but they are also quite simple to modify if you know where to look. 

For your specific problem, I just searched bubble-wrap.yaml for "piste" and I found this layer: https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L2232-L2283

When I tried changing [this line](https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L2252) to a value of "true", more slope labels appeared! If you're using a different scene file you should be able to follow a similar process. For a full explanation of the scene file syntax you can check out our docs site here: https://mapzen.com/documentation/tangram/Scene-file/

Hope this helps!

**Comment by yanickv** [comment]
Hi,

Works! With your suggestion I got the slope names to start appearing before level 15. But the google map is able to show more slope names, maybe because they show curved text that follow the slope shape. I have also tried the collide=false, but that was desastrous, the same ski slope name was showing many times on the same slope. 

![screenshot_2016-05-26-08-04-23](https://cloud.githubusercontent.com/assets/3969320/15574135/b4cf1b0a-2319-11e6-8d98-1d9eb8a67c0c.png)


For now I would say this is good enough for me. 

Thanks for the help!

Yanick
